src/printer.rs

```diff
@@ -124,18 +124,21 @@ impl Printer {
         theme: Option<Theme>,
         stream: bool,
         buffer: Buffer,
-        format_options: FormatOptions,
+        format_options: Vec<FormatOptions>,
     ) -> Self {
         let theme = theme.unwrap_or(Theme::Auto);
+        let format_options = format_options
+            .iter()
+            .fold(FormatOptions::default(), FormatOptions::merge);
```

Comment:
I think I'd rather do this in `main.rs`, as close to `cli.rs` as we can easily manage. It's not really the printer's responsibility.